spec/factories/users.rb

```diff
@@ -73,6 +73,25 @@
       end
     end
 
+    factory :unauthorized_evss_user, traits: [:loa3] do
```

Comment:
Instead of creating a whole new users would there be any advantage to making the evss (un)authorization into traits? Sort of like `:mhv_not_logged_in`? This file seems to include a fairly equal scattering of both approaches so I think it just comes down to which is most useful, in the absence of a clear convention.